projects/SCION/programs/basic/module.nix

```diff
@@ -0,0 +1,14 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+let
+  cfg = config.programs.scion;
+in
+{
+  options.programs.scion = {
+    enable = lib.mkEnableOption "scion";
```

Comment:
Can you follow the [new template](https://github.com/ngi-nix/ngipkgs/blob/fe2fd0e1fe05b07ea41be8498ae237d6f1cdfde7/templates/project/programs/_programName_/module.nix#L15-L18) and add packages from `projects-old/SCION-1M` here? After you finish, delete that directory and commit your changes.